### PR TITLE
Bumped NN, Maps, Common

### PR DESCRIPTION
**Pull request by VysotskiVadim**

### CODE REVIEW

**Review by Zayankovsky — APPROVED** [review]

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -13,17 +13,17 @@ ext {
...
 
   version = [
-      mapboxMapSdk              : '10.7.0-rc.1',
+      mapboxMapSdk              : '10.7.0',
       mapboxSdkServices         : '6.7.0-beta.1',
       mapboxEvents              : '8.1.5',
       mapboxCore                : '5.0.2',
       mapboxNavigator           : "${mapboxNavigatorVersion}",
-      mapboxCommonNative        : '22.1.0-rc.1',
+      mapboxCommonNative        : '22.1.0',
```
Comment (RingerJK) [inline gradle/dependencies.gradle:26]: nit: we need 22.1.1 to make the release

File: libnavigation-router/src/test/java/com/mapbox/navigation/route/internal/RouterWrapperTests.kt [hunk 2]
```diff
@@ -336,7 +336,8 @@ class RouterWrapperTests {
                 UUID,
                 1,
                 0,
-                RoutingProfile(routerOptions.profile().mapToRoutingMode(), routerOptions.user())
+                RoutingProfile(routerOptions.profile().mapToRoutingMode(), routerOptions.user()),
+                null
```
Comment (VysotskiVadim) [inline libnavigation-router/src/test/java/com/mapbox/navigation/route/internal/RouterWrapperTests.kt:340]: @DenisPryt, can you please confirm that if Android SDK passes null as current geometry index, route refresh doesn't change behaviour?

Comment (DenisPryt) [reply libnavigation-router/src/test/java/com/mapbox/navigation/route/internal/RouterWrapperTests.kt:340]: Yes, it doesn't change behaviour

File: gradle/dependencies.gradle (hunk 1 above)
Comment (VysotskiVadim) [reply gradle/dependencies.gradle:26]: will update it in separate PR if needed

**Pull request merged**